.github/workflows/ci.yml

```diff
@@ -0,0 +1,56 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+
+  build_kernel:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+
+      - name: Check out repository code
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: |
+          sudo apt update
+          sudo apt install -y build-essential gawk flex bison openssl dkms \
+          libelf-dev libudev-dev libpci-dev libiberty-dev autoconf dwarves \
+          libncurses-dev libssl-dev
+
+      - name: Generate .config 
+        run: |
+          rm -rf .git .gitattributes .gitignore
+          make defconfig
+          make syncconfig
+          make archprepare
+          ./scripts/kconfig/merge_config.sh .config ./scripts/package/truenas/tn.config
+          ./scripts/kconfig/merge_config.sh .config ./scripts/package/truenas/debug.config
```

Comment:
What do these do, merge .config into our local configs? Or the other way around?

Reply:
It's the other way around. We are merging tn.config and debug.config into .config.